scripts/Iris.json

```diff
@@ -15,45 +13,12 @@
                 "delimiter": ",",
                 "header_rows": 0
             },
-            "name": "Iris",
-            "schema": {
-                "fields": [
-                    {
-                        "name": "sepal_length",
-                        "type": "double"
-                    },
-                    {
-                        "name": "sepal_width",
-                        "type": "double"
-                    },
-                    {
-                        "name": "petal_length",
-                        "type": "double"
-                    },
-                    {
-                        "name": "petal_width",
-                        "type": "double"
-                    },
-                    {
-                        "name": "class",
-                        "size": "20",
-                        "type": "char"
-                    }
-                ]
-            },
-            "url": "http://mlr.cs.umass.edu/ml/machine-learning-databases/iris/iris.data"
-        }, 
-        {
-            "dialect": {
-                "delimiter": ",",
-                "header_rows": 0
-            },
             "name": "bezdekIris",
```

Comment:
Let's just call this Iris now since it's just the correct version of the Iris data.